sql-migration-assistant/src/sql_migration_assistant/infra/app_serving_cluster_infra.py

```diff
@@ -32,9 +32,7 @@ def choose_serving_cluster(self):
             cluster_name = self.cluster_name
             cluster_id = response.response.cluster_id
         else:
-            clusters = self.w.clusters.list(
-                filter_by=ListClustersFilterBy(cluster_states=[State.RUNNING])
-            )
+            clusters = self.w.clusters.list()
```

Comment:
I removed it because if the deployment goes wrong you cannot select the already running cluster from the last try but have to stop it or create a new one. this greatly increases the feedback loop